lang/en.js

```diff
@@ -111,7 +111,8 @@ export default {
 	"components.pageable.info": "{count, plural, one {{countFormatted} item} other {{countFormatted} items}}",
 	"components.pageable.info-with-total": "{totalCount, plural, one {{countFormatted} of {totalCountFormatted} item} other {{countFormatted} of {totalCountFormatted} items}}",
 	"components.pager-load-more.status-loading": "Loading more items",
-	"components.selection.action-hint": "Select an item to perform this action.",
+	"components.selection.action-max-hint": "You are over the selection limit of {countFormatted}.",
```

Comment:
I know the verbiage isn't final, but I think regardless it should be using the `plural` stuff here.